torchtitan/models/attention.py

```diff
@@ -26,6 +27,13 @@
 FLEX_ATTN_MASK_T = tuple[str, int | None]
 
 
+_flex_compile_mode: str | None = (
```

Comment:
Can we reuse `training.deterministic`? This is also a flag for debugging purpose IIUC. Using an environment variable needs user to read the code and understand FlexAttn compile mode.

Reply:
okay, this is a good point. Let me check. I just don't want to pass another config all the way to Attention.